installation/on_debian_and_ubuntu.md

```diff
@@ -40,3 +40,37 @@ When a new Crystal version is released you can upgrade your system using:
 sudo apt-get update
 sudo apt-get install crystal
 ```
+
+## Common Installation Issues on Debian Distro
+ 
+* #####Curl command not found
+	If curl is not already installed, then you might consider installing it using
+```
+sudo apt-get install curl
+```
+
+* #####The method driver /usr/lib/apt/methods/https could not be found
+
+	Debian package installation mechanism does not support HTTPS for its transport by default.To use HTTPS downloads, we have to install apt-transport-https . You can install the same using the following command:
+```
+sudo apt-get install apt-transport-https
+```
+
+* #####GPG Error that signatures can't be verified. Public key is not available.
```

Comment:
We can't recommend this in Crystal's documentation, it's asking people to trust any unsigned thing, which is a very bad practice. Have you tried the advice here? https://crystal-lang.org/docs/installation/on_debian_and_ubuntu.html#setup-repository

Reply:
I installed crystal in my system with the instructions from the official documentation but that error still showed up. apt-get update always returns with error of untrusted sources.